### PR TITLE
Fix Bundle: holosign creator, status procs, teleporter hub, timestop effect

### LINKED ISSUE #15910: Security Holobarrier Projector still counts destroyed barriers as projecting them
[Everyone]: # (Please fill in your information on the line AFTER the header.)

### LINKED ISSUE #15978: Handcuff and timestop bug
### What is the problem?

HELP: redacted (?) (PP) (VV) (SM) (FLW) (TP) (REJT): Found a bug. If you're handcuffed and get dragged into a TImestop, then you and the person dragging you can never move again, and you can't resist out of the cuffs. 

### Why is it a problem, or what should have happened?

Obvious exploit potential.

### How can the problem be reproduced?

By following the above procedure.

Other useful info: Summon magic was occurring at the time

### LINKED ISSUE #16857: Asleep alert not being cleared in a specific situation.
I got a heart attack ("Clayton Mull clutches at their chest as if their heart stopped!")
People gave me some pills and patches to heal the damages, not realising I had a heart attack probably.
I was put into a sleeper and then into cryo where I ultimately died.
Just before dying, I got a dream message ("... a blue light ...") suggesting I was sleeping, I don't think you're supposed to be considered asleep when you are actually in crit, so there's probably something wrong going on.
My corpse was ejected from cryo and defibbed, which revived me and cleared the heart attack. The chems in me healed the remaining damages, but I still had a "asleep" alert that wasn't cleared.
The Asleep alert remained even after reconnecting (despite my sleeping var being 0).
Only using the sleep command cleared the alert once I finished my nap.

### LINKED ISSUE #17031: alien/stun() proc missing key argument
https://github.com/tgstation/-tg-station/blob/98291ed10a60be5763b4e01ae96339f43bb077ee/code/modules/mob/living/carbon/alien/alien.dm#L140

it at least needs the argument and code to call update_canmove() or not. Also probably the argument to ignore the mob not having CANSTUN.

The proc should probably be moved in its own file with all other status procs (Weaken(), adjust_blindness(), etc..)

### LINKED ISSUE #17034: Teleporter Station doesn't have appropriate sprite when without power AND with its panel open
Also, all /obj/machinery/teleport machines are using stationobjs.dmi as their icon file instead of a proper file or at least something related to machines.

### PR DESCRIPTION
**Pull request by phil235**
- Fixes the holosign_creator's sign counter not going back up when one of its holosign is destroyed. Fixes #15910
- Fixes the lack of second argument in alien/stun() Fixes #17031
- Fixes the sleeping alert not being updated properly when a mob asleep is killed then revived (and same bug with druggy alert). Fixes #16857
- Fixes timestop effect leaving mobs unable to move forever in some cases. Fixes #15978
- Fixes teleporter hub and station staying active when the hub's panel is open, despite the hub's open panel sprite looking offline. The Hub and station now use their own iconfile icons/obj/machines/teleporter.dmi (similar what most machines have). Fixes #17034
- I've moved all mob's status adjustment procs (Stun(), Paralyse(), blur_eyes() adjust_drugginess(), etc...) into specific files (named "status_procs.dm") so they're easier to find ( @AnturK ). I've added a third argument to stun adjustments procs to ignore the CANSTUN flag. I've also changed the weaken procs to have the ignore_canweaken argument be the third argument, so all status adjustment procs have amount as first arg, updating the mob or not as second, and when necessary an ignore_x as third arg.

### CODE REVIEW

File: code/game/machinery/teleporter.dm [hunk 1]
```diff
@@ -450,9 +453,12 @@
 	if(stat & (BROKEN|NOPOWER) || !teleporter_hub || !teleporter_console )
 		return
 	if (teleporter_console.target)
-		src.engaged = !src.engaged
-		use_power(5000)
-		visible_message("<span class='notice'>Teleporter [engaged ? "" : "dis"]engaged!</span>")
+		if(teleporter_hub.panel_open || teleporter_hub.stat & (BROKEN|NOPOWER))
+			visible_message("<span class='alert'>The teleporter hub isn't responding.</span>")
```
Comment (phil235) [inline code/game/machinery/teleporter.dm]: can't engage the hub if it's without power, broken, or has its panel open.

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -1054,3 +932,15 @@ var/next_mob_id = 0
...
+
+/mob/living/carbon/alien/adjustToxLoss(amount)
+	return
+
+/mob/living/carbon/alien/adjustFireLoss(amount) // Weak to Fire
+	if(amount > 0)
+		..(amount * 2)
+	else
+		..(amount)
+	return
```
Comment (phil235) [inline code/modules/mob/mob.dm]: fuck

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 3]
```diff
@@ -160,7 +160,7 @@
 		M << "<span class='notice'>[high_message]</span>"
 	M.AdjustParalysis(-2, 0)
 	M.AdjustStunned(-2, 0)
-	M.AdjustWeakened(-2, 0, 0)
+	M.AdjustWeakened(-2, 0)
```
Comment (phil235) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: less ugly when all adjust proc use the same argument order.

**Comment by HankMoodyExperience** [comment]
can you also fix timestop so it isnt hot garbage

**Pull request merged**